### PR TITLE
Add test using Jest

### PR DESCRIPTION
**Pull request by XRichardroidX**
In this project, I
- Created a test file for add item and delete item
- Mock storage object to imitate localStorage
- Make sure tests are grouped using describe() method

### CODE REVIEW

File: __mocks__/userinterface.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+
+  it('Add another new task to the to-do list', () => {
+    // const tasksObject = new Tasks(2, 'Soccer', false);
+    UserInterface.addTask(tasksObject);
+    const mainChildLength = document.querySelector('main').children.length;
+    expect(mainChildLength).toBe(3);
+  });
+
+  it('Remove one task form list', () => {
```
Comment (VICTOR-CHUKWU) [inline __mocks__/userinterface.test.js]: - [ ] Good job so far, but I can not see any test for your `editTask` function, kindly add that

File: __mocks__/userinterface.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          </div>
+        </main> 
+    `;
+
+  const tasksObject = new Tasks(1, 'Exercise', false);
+
+  it('Add one new task to the to-do list', () => {
+    UserInterface.addTask(tasksObject);
+    const mainChildLength = document.querySelector('main').children.length;
+    // console.log(document.querySelector('main').children.length);
```
Comment (VICTOR-CHUKWU) [inline __mocks__/userinterface.test.js]: - [ ] kindly remove unused codes :+1:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @XRichardroidX  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @XRichardroidX👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**